src/appleseed.cli/stdouttilecallback.cpp

```diff
@@ -31,6 +31,7 @@
 
 // appleseed.renderer headers.
 #include "renderer/api/frame.h"
+#include "renderer/kernel/aov/imagestack.h"
```

Comment:
"External" applications such as appleseed.cli, appleseed.studio, etc. must only include API headers, in this case you want to include `renderer/api/aov.h`.